packages/resolver/lib/sources/npm.ts

```diff
@@ -74,6 +74,12 @@ export class NPM implements ResolverSource {
   // we're going to resolve it to some_module/contracts/AnotherContract.sol, ensuring
   // that when this path is evaluated this source is used again.
   resolveDependencyPath(import_path: string, dependency_path: string) {
```

Comment:
This is not a necessary one but if you feel like updating `dependency_path` to `dependencyPath` it would make me feel ever so slightly better :)

Reply:
Oh ha yes, easy change :)